src/Command/ErrorFormatter/OutputStyle.php

```diff
@@ -0,0 +1,101 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Command\ErrorFormatter;
+
+interface OutputStyle
+{
+	public const VERBOSITY_QUIET = 16;
+	public const VERBOSITY_NORMAL = 32;
+	public const VERBOSITY_VERBOSE = 64;
+	public const VERBOSITY_VERY_VERBOSE = 128;
+	public const VERBOSITY_DEBUG = 256;
+
+	public const OUTPUT_NORMAL = 1;
+	public const OUTPUT_RAW = 2;
+	public const OUTPUT_PLAIN = 4;
```

Comment:
I think I will removed this output levels as it is not really needed for error formatter use-case